tests/services/schemas/test_creator_contributor.py

```diff
@@ -205,14 +193,18 @@ def test_creator_invalid_identifiers_scheme():
         "family_name": "Cesar",
         "given_name": "Julio",
         "type": "personal",
-        "identifiers": {
-            "unapproved scheme": "0000-0002-1825-0097",
-        }
+        "identifiers": [{
+            "scheme": "unapproved scheme",
+            "identifier": "0000-0002-1825-0097",
+        }]
     }
 
     assert_raises_messages(
         lambda: CreatorSchema().load(invalid_scheme),
-        {'identifiers': ["Invalid value. Choose one of ['orcid', 'ror']."]}
+        {'identifiers': {0: {'_schema': [
```

Comment:
This might affect error handling on the UI, but is simply passing from dict to array (e.g. make use of `0` etc indexes).